package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "crafthead",
```

Comment:
It insists on doing this every time I build, usually, I discard the changes. Oh well, it's fine, just not under the purview of the PR.